site/src/components/BlogPosts.scss

```diff
@@ -0,0 +1,55 @@
+
+@import "../scss/init";
+
+.BlogPosts{
+  margin: rem-calc(24) 7.5%;
+  h2 {
+    padding-bottom: rem-calc(26);
+    background-image: url(../../src/img/down-chevrons.svg);
+    background-position: 50% 100%;
+    background-repeat: no-repeat;
+    text-align: center;
+    font-family: $body-font-family;
+    font-size: rem-calc(18);
+    line-height: 1.222222222;
+    font-weight: bold;
+  }
+  ul {
+    @include breakpoint(640px) {
+      display: flex;
+      justify-content: space-between;
+    }
+    margin: rem-calc(82) auto;
+    max-width: rem-calc(1280);
+    list-style: none;
+    li {
+      @include breakpoint(640px) {
```

Comment:
Didn't we adjust the medium breakpoint to 640px recently?  If so, can you use `@include breakpoint(medium)` instead of an explicit width?

Reply:
Correct! Will do.

Reply:
Also, I added 15px horizontal margins for increased consistency with both the INTRODUCTIONS section and the LCM site.